DESCRIPTION.md

```diff
@@ -8,12 +8,14 @@ Source code is also available at: https://github.com/snowflakedb/snowflake-conne
 
 # Release Notes
 
-- v3.0.1(unreleased)
+- v3.0.1(February 28, 2023)
 
-  - Improved the robustness of OCSP response caching to handle errors in cases of serialization and deserialization.
-  - Fixed a bug where `AuthByKeyPair.handle_timeout` should pass keyword arguments instead of positional arguments when calling `AuthByKeyPair.prepare`. PR #1440 (@emilhe)
+  - Improved the robustness of OCSP response caching to handle errors in cases of serialization and deserialization
```

Comment:
End each item with a period

Reply:
Historically we don't put periods at the ends, is that something that we should be doing from now on?